server/service/osquery.go

```diff
@@ -1405,3 +1411,139 @@ func (svc *Service) SubmitResultLogs(ctx context.Context, logs []json.RawMessage
 	}
 	return nil
 }
+
+////////////////////////////////////////////////////////////////////////////////
+// Query Reports
+////////////////////////////////////////////////////////////////////////////////
+
+const (
+	maxQueryReportRows = 1000
```

Comment:
I see this defined in two places, maybe it should be defined and exported in `server/fleet`?